content/shmem_team_split_2d.tex

```diff
@@ -38,22 +38,24 @@
   from \VAR{yaxis\_config}.}
 
 \apiargument{OUT}{yaxis\_team}{A new \ac{PE} team handle representing a \ac{PE}
-subset consisting of all the \acp{PE} that have the same coordinate along the \VAR{y}-axis
+subset consisting of all the \acp{PE} that have the same coordinate along the \VAR{x}-axis
 as the calling \ac{PE}.}
 \end{apiarguments}
 
 \apidescription{
-The \FUNC{shmem\_team\_split\_2d} routine is a collective routine. It creates two
-new teams by splitting an existing parent team into up to two subsets based on a
+The \FUNC{shmem\_team\_split\_2d} routine is a collective operation. It returns two
```

Comment:
Should we say `routine **performs** a collective operation`?

Reply:
The phrasing "is a collective" appears in 11 separate API descriptions and I found no instances of "performs a collective". Your phrasing is a bit better, but I think we should stick with consistency for now.

Reply:
We are not introducing the definitions for `operation` and `routine` this time, so I agree that being consistent would be the better option.